packages/eslint-config-airbnb/README.md

```diff
@@ -11,9 +11,9 @@ We export three ESLint configurations for your usage.
 ### eslint-config-airbnb
 
 Our default export contains all of our ESLint rules, including EcmaScript 6+
-and React. It requires `eslint` and `eslint-plugin-react`.
+and React. It requires `eslint`, `eslint-plugin-react` and `eslint-plugin-jsx-a11y`.
```

Comment:
i don't think it matters, the next line shows them being npm-installed, so you know where the page is